lib/hooks.ts

```diff
@@ -2,8 +2,8 @@ import useSWR from 'swr';
 import { useSession } from '../context/session';
 import { ErrorProps, ListItem } from '../types';
 
-async function fetcher(url: string, encodedContext: string) {
-    const res = await fetch(`${url}?context=${encodedContext}`);
+async function fetcher(url: string, encodedContext: string, query = '') {
+    const res = await fetch(`${url}?context=${encodedContext}${query}`);
```

Comment:
once we do the change below, I believe we'll need to add a `&` before query here